client/index.html

```diff
@@ -243,8 +243,9 @@ <h1 class="title">Settings</h1>
 							</div>
 							<div class="col-sm-12">
 								<label class="opt">
-								<input id="badge" type="checkbox" name="badge">
-								Enable badge
+								<input id="desktopNotifications" type="checkbox" name="desktopNotifications">
+								Enable desktop notifications<br>
+								<div class="error" id="warnDisabledDesktopNotifications">Warning : Desktop notifications are blocked by your web browser</div>
```

Comment:
:+1: on the space, that's a difference with French :P
For the warning, I don't think so at the moment, because none of the similar-looking error do that. I'd keep this for a later time.